src/e2e.test.ts

```diff
@@ -93,32 +94,34 @@ describe("End-to-end tests", () => {
 
   it("can differentiate between RDF and non-RDF Resources", async () => {
     const rdfResourceInfo = await fetchResourceInfoWithAcl(
-      "https://lit-e2e-test.inrupt.net/public/lit-pod-resource-info-test/litdataset.ttl"
+      `${rootContainer}lit-pod-resource-info-test/litdataset.ttl`
     );
     const nonRdfResourceInfo = await fetchResourceInfoWithAcl(
-      "https://lit-e2e-test.inrupt.net/public/lit-pod-resource-info-test/not-a-litdataset.png"
+      `${rootContainer}lit-pod-resource-info-test/not-a-litdataset.png`
     );
     expect(isRawData(rdfResourceInfo)).toBe(false);
     expect(isRawData(nonRdfResourceInfo)).toBe(true);
   });
 
   it("can create and remove empty Containers", async () => {
-    const newContainer1 = await createContainerAt(
-      "https://lit-e2e-test.inrupt.net/public/container-test/some-container/"
-    );
-    const newContainer2 = await createContainerInContainer(
-      "https://lit-e2e-test.inrupt.net/public/container-test/",
-      { slugSuggestion: "some-other-container" }
-    );
+    if (!rootContainer.includes("demo-ess")) {
```

Comment:
It's somewhat nitpicky, but I think it'd be good to add something to the output showing that this test was skipped.

One approach would be to include the entire `it()` in the `if` clause, and then in the `else` add:

```js
   it.todo("can create and remove empty Containers")
```

(Or alternatively, instead of `it` use `const nss = rootContainer.includes("demo-ess") ? it.skip : it`, and then do `nss("can create and remove empty Containers", async () => {` - so it is explicitly marked as skipped.)

This is assuming that Jest doesn't do weird magic that prevents you from actually passing around `it()`.

Reply:
Great idea, Jest doesn't complain about it: 876d84d8cddbe69167828c246f3e7b7d7e6eddb9

Reply:
Very cool.

![image](https://user-images.githubusercontent.com/4251/90528735-439be400-e173-11ea-9b45-ab4a6c37459b.png)

Reply:
(Very cool if you're a geek and interested in this sort of thing, I guess.)

Reply:
Who isn't ?